bpf/lib/lb.h

```diff
@@ -51,16 +56,75 @@ __BPF_MAP(cilium_lb4_services, BPF_MAP_TYPE_HASH, 0,
 	  sizeof(struct lb4_key), sizeof(struct lb4_service),
 	  PIN_GLOBAL_NS, CILIUM_LB_MAP_MAX_ENTRIES);
 
+__BPF_MAP(cilium_lb4_rr_seq, BPF_MAP_TYPE_HASH, 0,
+	  sizeof(struct lb4_key), sizeof(struct lb_sequence),
+	  PIN_GLOBAL_NS, CILIUM_LB_MAP_MAX_FE);
+
 #define REV_NAT_F_TUPLE_SADDR 1
+#ifdef LB_DEBUG
```

Comment:
cilium_trace should already be under DEBUG. If you enable all the DEBUGs the code will fail with instruction limit exceeded. So if I put it in #ifdef DEBUG it would mean we will not be able to load the code. That is why I put it in a separate debug. The only ones that are must are the 2 where I use cilium_trace. For remaining I use cilium_trace_lb which get commented out by default. They can be enabled and the thing loads fine, but if you disable Policy enforcement it fails. We need to debug that separately.